src/ess/sans/types.py

```diff
@@ -119,11 +119,31 @@ class TransmissionRun(sciline.Scope[ScatteringRunType, int], int):
 must be two wavelength values defining the start and end wavelength of the band."""
 
 
-QBins = NewType('QBins', sc.Variable)
-"""Q binning"""
+class QBins:
+    """Q binning, either 1-D (Q) or 2-D (Qx,Qy)."""
+
+    def __init__(
+        self,
+        bins: sc.Variable | None = None,
```

Comment:
```suggestion
        Q: sc.Variable | None = None,
```
? to be consistent with the others?

Reply:
It is a position-only arg, so the name does not matter (cannot be used as a keyword arg).